camera_type: "kinect_1"
full_path: "/home/ghanim/git/static_object_position_tracking/models/"
path_original_pcd: "/home/ghanim/git/static_object_position_tracking/src/"
path_filtered_pcd: "/home/ghanim/git/static_object_position_tracking/src/"
segm_limits: {x: {min: 0.45, max: 1.1}, y: {min: -0.6, max: 0.6}, z: {min: -0.13, max: 0.2}} ## normal table